metadat/hacks/Sega - Mega Drive - Genesis.dat

```diff
@@ -0,0 +1,40 @@
+clrmamepro (
+	name "Mega Drive hacks"
+	description "Romhacks of Sega Mega Drive / Sega Genesis games"
+)
+
+game (
+	name "Mega Man - The Wily Wars (Europe) Move-Shoot Hack"
+	description "Mega Man - The Wily Wars hacks for 60FPS, SRAM support and NES-like movement/shooting"
+	rom ( name "Mega Man - The Wily Wars (move-shoot hack).bin" size 2097152 crc 1d9b9c91 md5 c13a4784a8bce87bb514d6f24a05d2f1 sha1 99cb17c2b141c21b5661049d3811ebf90fde40b2 )
+)
+
+game (
+	name "Sonic 3 Complete (USA)"
+	description "Sonic 3 & Knuckles 3 hack version 130810"
+	rom ( name Sonic3C.gen size 3932160 crc 2bd564b1 md5 153b6d32026993569b16ff89c53197a3 sha1 4461ed4a94b93653905c08e201f7c11a128e5a47 )
+)
+
+game (
+	name "Street Fighter II' - Champion Edition Arcade (USA)"
+	description "Street Fighter II' - Special Champion Edition hack by Lord Hiryu"
+	rom ( name "Street Fighter II' - Arcade Champion Edition.bin" size 3145728 crc 6de3eea9 md5 4d808b2dc08eafee0f88a1ddb2fe9c28 sha1 dcaeba1b980ddab6cd51447b4f075ba63960fe34 )
+)
+
+game (
+	name "Street Fighter II' - Special Champion Edition (USA) PCM Driver Fix"
+	description "Street Fighter II' - Special Champion Edition with a rewritten Z80 PCM driver by Stef"
+	rom ( name sf2_mod_v7.bin size 3145728 crc 92b46306 md5 9b4529a4f713c733e5a187bc7b673eca sha1 1d6e640007edbec5e4e7a7d37d48e12a9029fa00 )
+)
+
+game (
+	name "Super Street Fighter II (USA) PCM Driver Fix"
+	description "Super Street Fighter II with a rewritten Z80 PCM driver by Stef"
+	rom ( name SSF2_mod.bin size 5242880 crc e8612c91 md5 3ed52bd4b63f0438ea41306aec08afcc sha1 479223c0efd45a1c88298dcd231af1c6f347f9f1 )
+)
+
+game (
+	name "Ultimate Mortal Kombat Trilogy (USA)"
+	description "Ultimate Mortal Kombat 3 hack by KABAL_MK version 5125"
+	rom ( name (5125)UMKT(HACK_23)_fixed.bin size 10485760 crc d083dbba md5 073bef4bea44531dd1a217e801ab04b7 sha1 140063d3a2cda353851c512571b6f706d0a96dfe )
```

Comment:
This filename looks pretty gross. Mind sticking it in quotes to make sure it doesn't break some things?